<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceAudioCO</class>
 <widget class="QDialog" name="InterfaceAudioCO">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonAppelProf">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSon">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>130</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="labelAppelProf">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Le prof a été appelé</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(0, 151, 178); /* Fond bleu */
    color: white; /* Texte en blanc */
    border: none; /* Enlever le contour si nécessaire */
    border-radius: 5px; /* Optionnel : coins arrondis */
    font-weight: bold; /* Optionnel : texte en gras */
}</string>
   </property>
   <property name="text">
    <string>Bienvenue sur l'interface pour une Compréhension Orale</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>800</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(0, 151, 178); /* Fond bleu */
    color: white; /* Texte en blanc */
    border: none; /* Enlever le contour si nécessaire */
    border-radius: 5px; /* Optionnel : coins arrondis */
    font-weight: bold; /* Optionnel : texte en gras */
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
